lte/gateway/python/magma/mobilityd/BUILD.bazel

```diff
@@ -0,0 +1,72 @@
+# Copyright 2021 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_python//python:defs.bzl", "py_binary", "py_library")
+load("@subpar//:subpar.bzl", "par_binary")
+load("@python_deps//:requirements.bzl", "requirement")
+
+DEBS = [
+    ":ip_address_man",
+    "//lte/protos:keyval_python_proto",
+    "//lte/protos:mconfigs_python_proto",
+    "//lte/protos:mobilityd_python_grpc",
+    "//lte/protos:mobilityd_python_proto",
+    "//lte/protos:subscriberdb_python_grpc",
+    "//lte/protos:subscriberdb_python_proto",
+    "//orc8r/gateway/python/magma/common:metrics",
+    "//orc8r/gateway/python/magma/common:sentry",
+    "//orc8r/gateway/python/magma/common/redis:client",
+    "//orc8r/gateway/python/magma/configuration:service_configs",
+    "//orc8r/protos:eventd_python_grpc",
+    "//orc8r/protos:mconfig_python_proto",
+    "//orc8r/protos:mconfigs_python_proto",
+    "//orc8r/protos:metricsd_python_proto",
+    "//orc8r/protos:redis_python_proto",
+    "//orc8r/protos:service303_python_grpc",
+    "//orc8r/protos:service303_python_proto",
+]
+
+par_binary(
+    name = "mobilityd_par",
+    srcs = glob(["*.py"]),
```

Comment:
Here and elsewhere, glob use is helpful for prototyping but is generally discouraged when you finalize the work.  Probably this could use a brief explanation but I'll try to do it longer-form here in text.

Generally you want to avoid Bazel labels that contain vast swaths of files, unless they are forced by e.g. cyclic includes (bazel does not allow for cycles in dependencies between labels).  Why? A] Because this allows Bazel to rebuild the smallest possible component upon any change / build & run.  I don't know if this is a lesser issue for Python, cannot speak from experience there. But in Go / C++ this allows far smaller re-build components of the build graph.  B] This allows cache contents to be maximally re-used over time (e.g. say we sync a bazel cache weekly from CI phase for new contributors - that cache will be valid for any labels that don't themselves - or their dependencies - have any changes since the cache was built). C] Bazel only makes visible / available the sources explicitly enumerated.  By writing each source out we ensure that authors are truly aware of what files are actually needed.  There may be D] but I'm going to move on, let me know if you have thoughts / questions! Again using glob for prototyping is totally reasonable (I've done so myself many times).

In this particular case, you may find yourself wanting to break apart some of this python into py_libraries that then get dep'd into this par_binary (which per comment above about deprecation of par_binary, may actually be just a py_binary)?

Reply:
> Generally you want to avoid Bazel labels that contain vast swaths of files [...]. Why?
> 1. Because this allows Bazel to rebuild the smallest possible component upon any change / build & run. I don't know if this is a lesser issue for Python, cannot speak from experience there. But in Go / C++ this allows far smaller re-build components of the build graph.
> 1. This allows cache contents to be maximally re-used over time (e.g. say we sync a bazel cache weekly from CI phase for new contributors - that cache will be valid for any labels that don't themselves - or their dependencies - have any changes since the cache was built).
> 1. Bazel only makes visible / available the sources explicitly enumerated. By writing each source out we ensure that authors are truly aware of what files are actually needed.

In general, I wonder, if this would be a topic best done by the according teams for each service during or post migration, as they know the dependecy graph of the respective services best and it might be easier for them to write it down explicitly?

> Again using glob for prototyping is totally reasonable (I've done so myself many times).

😄 As you said, using glob is a big help when prototyping, so we thought: "Make it work first, make it nice afterwards." 😇 

For this PR, we introduced a list of files anyway. ✔️

Reply:
This means that any new file will have to be included in the Bazel config manually? I think this might feel rather unexpected and inconvenient from a Python developer's point of view.